src/components/RadioGroup/RadioGroup.jsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import FormLabel from '../FormLabel/FormLabel';
+import InputValidationMessage from '../InputValidationMessage/InputValidationMessage';
+import './RadioGroup.scss';
+
+/**
+ * Allows users to pick a value from predefined list of options.
+ * Ideally, it should be used when there are less than 5 options,
+ * otherwise you should consider using SelectInput instead.
+ */
+
+const propTypes = {
+  /**
+   * Radio group 'name'
+   */
+  name: PropTypes.string.isRequired,
+  /**
+   * Callback function to call on change event.
+   */
+  onChange: PropTypes.func.isRequired,
+  /**
+   * Options for radio group
+   */
+  options: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    value: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+    isDisabled: PropTypes.bool,
+  })).isRequired,
+  /**
+   * Custom content to be displayed above the RadioGroup.
+   * Both a title and description can be included in the legend.
+   */
+  legend: PropTypes.shape({
+    title: PropTypes.node.isRequired,
+    description: PropTypes.node,
```

Comment:
Instead of a `legend` object prop, can we have separate `title` and `description` props? Makes markup simpler on the consumer end.

Reply:
hmmm... reading more of your comments, now I understand better what you are doing with the legend object with a required `title`.

I guess I prefer the latter when it comes to actually using the component
```
<RadioGroupInput
  legend={{ 
    title: 'hello world',
    description: 'the description'
  }}
...
/>

vs

<RadioGroup
  title="hello world"
  description="the description"
  ...
/>
```

If you pass a description without a title, you're still going to get a propType warning.